src/extension.ts

```diff
@@ -26,6 +27,23 @@ export function activate(context: vscode.ExtensionContext) {
   controller.refreshHandler = (token: vscode.CancellationToken) =>
     ctest.refresh_tests(controller, log_channel, token);
 
+  const runProfile = controller.createRunProfile(
+    "Run",
+    vscode.TestRunProfileKind.Run,
+    (request, token) => {
+      run_tests(controller, log_channel, false, request, token);
```

Comment:
Using `false` like this is similar to using a magic number. Another developer needs to find `run_tests()` to understand what `false` means here. Before line 30, declare a constant like so:

```ts
const run_debug = true;
const runProfile = controller.createRunProfile(
  "Run",
  vscode.TestRunProfileKind.Run,
  (request, token) => {
    run_tests(controller, log_channel, !runDebug, request, token);
  }
);
```

We could go with `const no_debug = false;`, but I typically assume a boolean variable name represents the `true` state. Hence using `const run_debug = true;`, then negating it in the function call.

Reply:
Sounds good, we should change the saying to "don't use magic literals"